features/appearance.yml

```diff
@@ -2,3 +2,18 @@ name: appearance
 description: "The `appearance` CSS property controls the appearance of form controls. Using `appearance: none` disables any default native appearance and allows the elements to be styled with CSS."
 spec: https://drafts.csswg.org/css-ui-4/#appearance-switching
 group: css
+compat_features:
```

Comment:
I undisted this feature because the BCD tag now also contains the new `css.properties.appearance.base` which made this feature lose its baseline status.
Undisting the feature makes it possible to keep the same BCD keys as before the move to BCD 5.7.2 and move the new `base` value to the customizable-select feature instead.